kod/object/item/passitem/chalice.kod

```diff
@@ -76,15 +95,22 @@ messages:
       }
 
       % If we drop it in a room with a high Shal'ille bonus, refill the chalice
+      % And inform the last owner (if any) that the chalice has been refilled
```

Comment:
Fix grammar (lowercase And?  Two sentences with periods?)

Reply:
I tend to be a bit lazy with comments and grammar, but consider it done. :)